README.md

```diff
@@ -519,3 +519,176 @@ You can omit this argument to ignore this filter.
 
 This is the `reference` you gave at the time of sending the notification. The `reference` can be a unique identifier for the notification or an identifier for a batch of notifications.
 
+## Get a template by ID
+
+```
+response = notifications_client.get_template(
+    'template_id'
+)
+```
+
+<details>
+<summary>
+Response
+</summary>
+
+If the request is successful, `response` will be a `dict`:
+
+```python
+{
+    "id": "notify_id", # required
+    "type": "sms" | "letter" | "email", # required
+    "created_at": "created at", # required
+    "updated_at": "updated at", # required
+    "version": "version", # required
+    "created_by": "created by", # required
```

Comment:
checking, it's an email. you should probably put a mock email in, like `"created_by": "person@example.gov.uk"` or something